src/routes/api/trips.routes.js

```diff
@@ -228,7 +228,46 @@ tripsController.rejectTrip);
  *         description: Pending requests retrieved successfully
  *  */
 router.get('/avail-requests', checkUser, isProfileUpdated, validateParams, tripsController.availRequests);
-
+/**
+ * @swagger
+ *
+ * /api/v1/trips/most-traveled-centres?page={page}&limit={limit}:
+ *   get:
+ *     security: []
+ *     summary: Get most traveled destinations
+ *     description: Returns infos about most traveled centres
+ *     tags:
+ *       - Accommodations
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: integer
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: array
+ *     parameters:
+ *       - name: page
+ *         description: page number.
+ *         in: path
+ *         required: false
+ *         default: 1
+ *         type: string
+ *       - name: limit
+ *         description: limited items.
+ *         in: path
+ *         required: false
+ *         default: 2
+ *         type: string
+ *     responses:
+ *       200:
+ *         description: Trending centres are retrieved successfully
+ *  */
+router.get('/most-traveled-centres', [checkUser, isProfileUpdated],
```

Comment:
is it possible that you put this in the accommodation routes

Reply:
@blaise82 Thank you very much for that. I fixed it